### ISSUE TITLE
Why is light theme background so different from original?

### ISSUE THREAD

**Issue by heralden** [pappasam/papercolor-theme-slim#1]
Just a simple question. When comparing the screenshot of the light theme from the readme, the background has a yellow tinge compared to the original papercolor theme from https://github.com/NLKNguyen/papercolor-theme

I've been a big fan of papercolor light for years and want to try out treesitter - why is the background colour used here so different?

**Comment by pappasam** [comment]
@uosl it was my personal preference. That said, I realized that I should probably implement my personal preference as an override in my own vimrc instead of making it the default for everyone. I've changed the default light background color to be the same as papercolor-theme and documented how a user might override this in their vimrc. Thanks for raising this issue, we can consider this resolved now!
